uwblueprint/focus-on-nature
# feat: add waitlisted campers to get camper api

## Notion ticket link

[Ticket Name](https://uwblueprintexecs.notion.site/Add-waitlistedCampers-to-get-campers-48ca0eddd69140b8a2c5fe02e4b04c10)



## Implementation description
* Add waitlisted campers to get campers by id request



## Steps to test
1. call api endpoint for getting campers by camp id
2. ensure data comes back in correct schema



## What should reviewers focus on?
* ensure correct schema


## Checklist
